### Варіант № 3
### Завдання № 517(д)
Дано файл $f$, який містить відомості про іграшки: вказується назва
іграшки (наприклад, лялька, кубики, м$'$яч, конструктор і т. д.), її
вартість в копійках і можливі вікові межі дітей, для яких іграшка
призначена (наприклад, для дітей від двох до п$`$яти років). Отримати
ціни всіх кубиків оформленні по зразку ...грн ...коп.

In [2]:
with open('20_f.txt', encoding = 'utf-8') as f:
    data_f = f.read(); f.close()
list_f = data_f.split('\n')
list_f = [list_f[i].split(', ') for i in range(len(list_f))]
print(data_f, "\n\n", [f"{s[1][:-2]} грн {s[1][-2:]} коп"
                       for s in list_f if ("кубики" in s[0])])

конструктор металевий, 21689, від 12 до 16 років
кубики пластмасові, 11267, від 3 до 6 років
рація, 36850, від 14 до 18 років
вантажівка, 23600, від 10 до 16 років
кубики з літерами, 7640, від 3 до 9 років
конструктор пластмасовий, 9900, від 6 до 12 років
радіомашинка, 36720, від 16 до 18 років
дартц, 25000, від 16 до 99 років
кубики з магнітними вставками, 47900, від 13 до 18 років
модель автомобіля, 5910, від 9 до 18 років 

 ['112 грн 67 коп', '76 грн 40 коп', '479 грн 00 коп']


# Examples

### Example 1
конструктор металевий, 21689, від 12 до 16 років
кубики пластмасові, 11267, від 3 до 6 років
рація, 36850, від 14 до 18 років
вантажівка, 23600, від 10 до 16 років
кубики з літерами, 7640, від 3 до 9 років
конструктор пластмасовий, 9900, від 6 до 12 років
радіомашинка, 36720, від 16 до 18 років
дартц, 25000, від 16 до 99 років
кубики з магнітними вставками, 47900, від 13 до 18 років
модель автомобіля, 5910, від 9 до 18 років

['112грн 67коп', '76грн 40коп', '479грн 00коп']